### DISCUSSION TITLE
How can I create a composite model?

### DISCUSSION THREAD

**Discussion by michael-mertens** [TabularEditor/TabularEditor3#964] [🙋♀️TE3 Q&A]
Hi, 

I don't find anything about it in the documentation, but I hope it is possible ...
How can I create a composite model. I thought I would start with a standard import model and then would have the chance to add an additional analysis services data source in direct query mode.

I hope that I'm just doing it in the wrong way and it is possible at all.

Thanks & Regards
Michael

**Comment by otykier** [comment]
Unfortunately, we don’t currently have native support for importing tables from other AS models/PBI datasets, so your best option at the moment, is to create the composite model in PBI Desktop, then continue the work in Tabular Editor. When you need to add additional DQ over AS tables, you can once again create them using Desktop, then copy/paste using Tabular Editor.

We are working on providing a much better experience for this out of the box in TE3. Stay tuned!

**Comment by michael-mertens** [comment]
Ok, thank you, Daniel
